text/safrole.tex

```diff
@@ -81,7 +81,7 @@ \subsection{Key Rotation}\label{sec:keyrotation}
   \lambda \in \seq{\mathbb{K}}_{\mathsf{V}}
 \end{align}
 
-We must introduce $\mathbb{K}$, the set of validator key tuples. This is a combination of cryptographic public keys for Bandersnatch and Ed25519 cryptography, and a third metadata key which is an opaque octet sequence, but utilized to specify practical identifiers for the validator, not least a hardware address.
+We must introduce $\mathbb{K}$, the set of validator key tuples. This is a combination of a set of cryptographic public keys and metadata which is an opaque octet sequence, but utilized to specify practical identifiers for the validator, not least a hardware address.
```

Comment:
Now there is also the BLS key. The keys set content is further elaborated just below